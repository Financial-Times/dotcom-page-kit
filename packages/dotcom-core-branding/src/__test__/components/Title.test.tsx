import React from 'react'
import renderer from 'react-test-renderer'
import Subject from '../../components/Title'

describe('dotcom-ui-shell/src/components/Title', () => {
  it('renders the document title when pageTitle is passed', () => {
    const tree = renderer.create(<Subject pageTitle="test title" />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders the document title when siteTitle is passed', () => {
    const tree = renderer.create(<Subject pageTitle="test page title" siteTitle="test site title" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
