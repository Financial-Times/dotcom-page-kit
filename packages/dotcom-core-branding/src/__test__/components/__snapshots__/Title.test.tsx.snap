// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dotcom-ui-shell/src/components/Title renders the document title when pageTitle is passed 1`] = `
<title>
  test title | Financial Times
</title>
`;

exports[`dotcom-ui-shell/src/components/Title renders the document title when siteTitle is passed 1`] = `
<title>
  test page title | test site title
</title>
`;
